elrond-wasm/src/contract_base/wrappers/call_value_wrapper.rs

```diff
@@ -58,10 +58,12 @@ where
     /// Warning, not tested with multi transfer, use `all_esdt_transfers` instead!
```

Comment:
You made this content deprecated, so can be changed.
Point out that this crashes for multi-transfer.